cmd/docker-generate/manifest_test.go

```diff
@@ -0,0 +1,69 @@
+package main
```

Comment:
This is new to me and sounds like a good way to do unit testing on flag/argument parsing behavior. @shizhMSFT  Do we need to apply this to oras-project/oras?